### DISCUSSION TITLE
Can I Start to learn react with some knowledge of Javascript

### DISCUSSION THREAD

**Discussion by evansso** [community/community#60481] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

i want to learn react and also javascript.what should i do for now . at javascript i have learned conditionals scope functions data types variables. what advice can you give me

**Comment by nirav-gajera** [comment]
1. Expand your JavaScript knowledge: Since React is built with JavaScript, it's important to have a strong foundation in JavaScript. Continue learning JavaScript by exploring more advanced topics such as object-oriented programming, arrays, loops, error handling, and asynchronous programming (Promises, async/await).
2. Dive into React fundamentals: Start by learning the core concepts of React, including components, JSX syntax, props, state, and lifecycle methods. You can find numerous online tutorials, documentation, and video courses that cover React fundamentals.
3. Explore React ecosystem and libraries: As you progress, explore the broader React ecosystem and popular libraries and tools that complement React development, such as React Router for routing, Redux for state management, and Axios for making API requests.
4. Read official documentation: Refer to the official React documentation (https://reactjs.org/docs/) to gain a deeper understanding of React's concepts and features. It provides comprehensive explanations and examples that can help you solidify your knowledge.

**Comment by sumingcheng** [comment, marked as answer]
If you're learning JavaScript, try to create a small demo using what you've learned as much as possible. If you're not sure what kind of demo to implement, clone some interesting JavaScript projects on GitHub. The same goes for learning React. Don't let the language limit you. Always keep in mind the reason why you're learning.

**Comment by MUK-Dev** [comment]
Yes, you can.

I suggest learning about [es6 standards](https://www.w3schools.com/js/js_es6.asp) to JavaScript 2022 standards. These will really help you with React.
After these, you should learn React and start small by creating projects.
An excellent free crash course is from [Academind](https://www.youtube.com/watch?v=4baq00tHfmA)
